### PR TITLE
Issue #2

### PR DESCRIPTION
**Pull request by Ab-hay**
*Issue #, if available:*
#2

### CODE REVIEW

File: browser/components/ContributionsForm.tsx [hunk 1]
```diff
@@ -83,10 +83,8 @@ class ContributionsForm extends React.Component<Partial<Props>, State> {
       const approvers = this.props.approvers;
       return approvers.map(listValue => {
         return {
-          label: listValue.approver_name
-            ? listValue.approver_name
-            : listValue.approver_alias,
-          value: listValue.approver_id,
+          label: listValue,
+          value: listValue,
```
Comment (jpeddicord) [inline browser/components/ContributionsForm.tsx:87]: What changed here? I don't see where `props.approvers` is different. Or did this never actually work?

**Review by jpeddicord — APPROVED** [review]
Looks good! Seems to work well when I play around with it. I had one comment about the content of the select elements; if that's a non-issue then feel free to merge this.

File: browser/components/ContributionsForm.tsx (hunk 1 above)
Comment (Ab-hay) [reply browser/components/ContributionsForm.tsx:87]: This wasn't getting used at all previously. react-select need data in form of {label: value}.

**Comment by jpeddicord** [comment]
Merged in 75776be; branch included additional changes so closing this to open a new PR later.

**Pull request closed without merging**